bluesky_queueserver/manager/tests/_common.py

```diff
@@ -83,28 +87,133 @@ def patch_first_startup_file_undo(pc_path):
         os.rename(fln_tmp, fln)
 
 
+class TinyZmqClient:
+    """
+    This is a simplified version of ZMQ Client, which is very similar to the client
+    used in ``qserver_cli``. The server does not perform any verification of input
+    and output data and intended for testing 0MQ API of RE Manager.
+    """
```

Comment:
Would it make sense to centralize all 0MQ clients in one place? It feels like a duplicate functionality reimplemented in a few places (http server, qserver_cli, etc.)

Reply:
It's a good question. I think that with time functionality of the calls made via server will change and the tests will change. I don't think that duplicate testing is a problem right now. Also the functioning of the CLI and REST API has to be checked with different combinations of parameters as well. I think that CLI interface has to be changed also to be more user friendly, so the tests will change.

Reply:
OK, let's keep it in mind in the future, and keep it as is for now.